lib/web/requirejs/require.js

```diff
@@ -1445,25 +1445,18 @@ var requirejs, require, define;
                     //Grab defines waiting in the global queue.
                     intakeDefines();
 
-                    //Mark all the dependencies as needing to be loaded.
```

Comment:
@AndresInSpace, since this is the change in requires lib, I think you should create this pull request in https://github.com/requirejs/requirejs. And then just update the lib here

Reply:
https://github.com/requirejs/requirejs/pull/1859